assets/js/collaborative-editor/stores/createWorkflowStore.ts

```diff
@@ -1198,7 +1198,11 @@ export const createWorkflowStore = () => {
    * @param errors - Field errors { fieldName: ["error1", "error2"] }
    *                 Empty array [] clears that field
    */
-  const setClientErrors = (path: string, errors: Record<string, string[]>) => {
+  const setClientErrors = (
+    path: string,
+    errors: Record<string, string[]>,
+    isServerUpdates: boolean = false
```

Comment:
I note that we have elsewhere `isServerUpdate` (singular). Is this variable name with `s` at the end a typo ?